dfget/core/uploader/uploader.go

```diff
@@ -263,7 +264,7 @@ func serverGC(cfg *config.Config, interval time.Duration) {
 	}
 }
 
-func deleteExpiredFile(api api.SupernodeAPI, path string, info os.FileInfo,
+func deleteExpiredFile(p2p *peerServer, path string, info os.FileInfo,
```

Comment:
I am sure that this function works fine for us. While I am wondering if we could encapsulate this function to be one part of `peerServer`, rather than making `peerServer` a passed parameter for this function.

Then I think, with the change, it would be like `func (p2p *peerServer) deleteExpiredFile(path string, info os.FileInfo, expireTime time.Duration) bool)`.

WDYT? @lowzj

Reply:
OK, I will rewrite this.